app/src/main/assets/app/index.js

```diff
@@ -30,6 +30,10 @@ const secondFunction = require("./subdirectory/index")
 
 secondFunction()
 
+setInterval(function() {
+    console.log("timer fire")
+}, 2000)
```

Comment:
@patgoley did we mean to leave this in there?

Reply:
@microchip128 Yeah, it's ok I think. This is inside the example application which just loads some javascript to show it work.